Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -183,7 +183,9 @@ void Font::platformInit()
 #endif
 
     // Compute line spacing before the line metrics hacks are applied.
+#if !PLATFORM(IOS_FAMILY)
     float lineSpacing = lroundf(ascent) + lroundf(descent) + lroundf(lineGap);
```

Comment:
The use of `lroundf` here is a bug dating back to when `CGFloat` was 32-bit, causing these numbers to all be converted to `float` before being rounded. It would be better to use `std::lround` here, and fixing it seems low risk.